brainstormforce/astra
# AST-2231 - Gutenberg editor padding not applied for heading blocks issue

### Description

- BugFix: There should not be any padding, and it should look the same as in backend editor.
### Screenshots

Test Screencast: [Screencast at September 23rd 2022 - 2.mp4](https://d.pr/v/eb1X3X)

Note: please test with 3-4 different templates.
### Types of changes


- Gutenberg css specificity increased so that padding is applied correctly.



### How has this been tested?

- On a fresh install, installed Astra, Astra Addon, Spectra.
- Created new page with Gutenberg editor.
- Now take any heading block h1, h2, h3, and so on.
- Apply background color from the settings.
- Background color will be added to heading with appropriate padding.
- Check in frontend, same will be visible.

### Checklist:
